official/docs/node/current/claims/cancel.js

```diff
@@ -0,0 +1,9 @@
+const EasyPostClient = require('@easypost/api');
+
+const client = new EasyPostClient('EASYPOST_API_KEY');
+
+(async () => {
+  const claim = await client.Claim.refund('clm_...');
```

Comment:
The node library doesn't actually have a `refund`. It is instead `cancel`
```suggestion
  const claim = await client.Claim.cancel('clm_...');
```